tsconfig.json

```diff
@@ -15,6 +15,7 @@
       "*": ["node_modules/*"]
     },
     "allowSyntheticDefaultImports": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
Using `error` defined in catch clauses without narrowing or checking its type is now an error in typescript, making the build fail. I preferred disabling this instead of fixing this issue in a ton of files.

Reply:
Could you create a card/issue to follow up on this?

Reply:
Done 👍